src/styles/hamburger.css

```diff
@@ -0,0 +1,66 @@
+.bm-burger-button {
+    position: fixed;
+    width: 30px;
+    height: 25px;
+    top: 30px;
+    right: 30px;  
+}
+@media screen and (min-width: 768px) {
+    .bm-burger-button, .bm-menu-wrap {
+        display: none;
+    }
+}
+.bm-burger-bars {
+    background: #a6a6a6;
```

Comment:
Add css color variables so we know which colours these are but should move to JS anyways